fic_modules/markdown_modules.py

```diff
@@ -71,10 +71,11 @@ def create_git_md_table(repository_name, path_to_files):
                 commit_author = data.get(key).get("commiter_name")
                 commit_author = re.sub("\u0131", "i", commit_author)
                 date = data.get(key).get("commit_date")
+                url = data.get(key).get("url")
```

Comment:
this URL had to exist before calling trim_commit_description() function so I moved it up a few lines.